stable/nats/templates/NOTES.txt

```diff
@@ -0,0 +1,76 @@
+** Please be patient while the chart is being deployed **
+
+{{- if or (contains .Values.clientService.type "LoadBalancer") (contains .Values.clientService.type "nodePort") }}
+{{- if not .Values.auth.enabled }}
+{{ if and (not .Values.networkPolicy.enabled) (.Values.networkPolicy.allowExternal) }}
+
+-------------------------------------------------------------------------------
+ WARNING
+
+    By specifying "clientService.type=NodePort/LoadBalancer" and "auth.enabled=false"
+    you have most likely exposed the NATS service externally without any authentication
+    mechanism.
+
+    For security reasons, we strongly suggest that you switch to "ClusterIP". As
+    alternative, you can also switch to "auth.enabled=true" providing a valid
+    password on "auth.password" parameter.
+
+-------------------------------------------------------------------------------
+{{- end }}
+{{- end }}
+{{- end }}
+
+NATS can be accessed via port {{ .Values.clientService.port }} on the following DNS name from within your cluster:
+
+   {{ template "nats.fullname" . }}-client.{{ .Release.Namespace }}.svc.cluster.local
+
+{{- if .Values.auth.enabled }}
+To get the authentication credentials, run:
+
+    export NATS_USER=$(kubectl get cm --namespace {{ .Release.Namespace }} {{ template "nats.fullname" . }} -o jsonpath='{.data.*}' | grep -m 1 user | awk '{print $2}')
+    export NATS_PWD=$(kubectl get cm --namespace {{ .Release.Namespace }} {{ template "nats.fullname" . }} -o jsonpath='{.data.*}' | grep -m 1 password | awk '{print $2}')
```

Comment:
can you use `NATS_PASSWORD` or `NATS_PASS`? `NATS_PWD` is confusing to me because of the `pwd` linux command

Reply:
All right